docs/release.md

```diff
@@ -20,41 +23,52 @@ release, those are:
 These artifacts are provided in their respective channels and their natural way
 of distribution.
 
-# Cadence and Timeline
 
-# Minor releases
+# Platform Support
 
-KubeVirt aligns to the [Kubernetes release cycle](https://www.kubernetes.dev/resources/release/).
-But because KubeVirt builds upon Kubernetes, it will trail Kubernetes' release schedule by roughly 2 months, in order to have time to consume and stabilize on top of new releases.
+Every KubeVirt release is targeted towards one specific Kubernetes release.
+This is reflected in three areas:
 
-Thus new release branches are cut 3 times a year, or about every 15 weeks.
+1. _CI_ - A precondition of a release is the presence of CI lanes for the targeted Kubernetes release
+2. _Schedule_ - A KubeVirt release will be trailing it's corresponding Kubernetes release. It is trailing it, in order to perform the stabilization on a released Kubernetes version
+3. _Support matrix_ - A KubeVirt release's maintenance phase is tied to the maintenance phase of a Kubernetes release.
 
-## Guiding principles
+The primary reasons for defining a target Kubernetes release are:
 
-The release schedule is build around a few guiding principles:
+- CI resources are finite. And people to maintain it as well.
+- Limit the maintenance burden
+- Understanding the target platform in order to perform proper integration work
+- Setting the right expectations with end-users
 
-- Alpha and Beta tags are applied on main
-- Release branches must only be cut after a Kubernetes minor releases
-- RCs must only be tagged on the release branch
+The Kubernetes release corresponding to a KubeVirt release is defined in the release metadata of a KubeVirt release maintained by [SIG Release](https://github.com/kubevirt/sig-release/).
 
-## CI Provider
+## Platform Support Skew
 
-A pre-condition for the GA of a KubeVirt release is the presence of a KubeVirt CI provider for the corresponding Kubernetes release.
-Around the beta release of the corresponding Kubernetes release, the KubeVirt team will start to work on a KubeVirt CI provider.
-The assumption is, that the KubeVirt CI provider will be available by KubeVirt's enhancement freeze.
+While there are good reasons for defining a target Kubernetes release there are usually also needs to support more than one Kubernetes version.
+While the KubeVirt Community can not procide this support, patches to support other releases will also not be rejected as long as they are provided in a reasonable manner (scope and time wise).
```

Comment:
`While the KubeVirt Community can not procide this support` oh interesting. So are we saying we are not supporting N-2 k8s compatiblity?!

Reply:
We support KubeVirt `N` _on_ Kubernetes `tN`.
And then we support this tuple for the support scope of Kuebrnetes, and thus the support scope of KubeVirt (KubeVirt's scope is tied to Kubernetes support scope).

We would *not* support KubeVirt `N` on Kubenetes`tN-2`

(`N`: A kubeVirt version, `tN` target Kubernetes version of a KubeVirt version)

Reply:
This seems to me like a new policy differing from what we did provide until now. Please correct me if I am misunderstanding this.

Reply:
I've added an example.